### ISSUE TITLE
[terminal] The terminal scroll-bar is hardly visible

### ISSUE THREAD

**Issue by kittaakos** [eclipse-theia/theia#8096]
### Bug Description:

Check the scroll-bar in the terminal, it is hardly visible compared to the scroll-bar in the editors. It must be a UI regression.

Gitpod:
![gitpod-scrollbar](https://user-images.githubusercontent.com/1405703/85832327-27da1a00-b790-11ea-84b1-5df8fedcb7ab.gif)

Local (browser):
![terminal-scrollbar](https://user-images.githubusercontent.com/1405703/85832348-3294af00-b790-11ea-8e26-dd843797231f.gif)

### Steps to Reproduce:

1.
2.
3.

### Additional Information

- Operating System:
- Theia Version:

**Comment by tsmaeder** [comment]
@vince-fugnitto there are scroll bars on the terminal? ;-)

I am not sure whether the "no scroll bar" and the "small scroll bar" issue are the same thing. Do you have info not in the issues that makes you think so? And anyway....the impact of "no scroll bar" on usability is much greater, so I would rather have closed this issue than the other.

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto there are scroll bars on the terminal? ;-)

@tsmaeder yes, given that there is enough output in the terminal (ex: multiple `ls -al` performed), then a scrollbar becomes visible. The issue is that the scrollbar is quite small. 

Did you encounter an issue where the scrollbar was not visible at all? If so then we can re-open https://github.com/eclipse-theia/theia/issues/1954 provided we have steps to reproduce.

**Comment by tsmaeder** [comment]
I regularly see _no_ scrollbar, but that's in Che. I'll test in pure Theia when I get a moment.

**Comment by tsmaeder** [comment]
That was fast! Here's a screenshot without a scrollbar. This is just plain Theia master on FF/Windows.

![NoScrollbar](https://user-images.githubusercontent.com/13163770/108724548-b96b7880-7525-11eb-89ae-1643b0c0d265.PNG)

**Comment by vince-fugnitto** [comment]
@tsmaeder thanks for pointing it out, it seems to be an issue with Firefox (I was also able to reproduce on Linux) :+1:

**Comment by kittaakos** [comment]
From [here](https://github.com/eclipse-theia/theia/issues/12911#issue-1898537530) with the steps to reproduce:

> ### Bug Description:
> When I resize the terminal, the width of the vertical scrollbar changes; sometimes, it's hardly visible.
> 
> ### Steps to Reproduce:
> 1. Open a terminal and make sure you can vertically scroll in it (press `Enter` a few times, for example)
> 2. Check the width of the scrollbar
> 3. Horizontally resize the terminal widget to see that the width of the scrollbar changes.
> 
>  dynamic_scrollbar_width_in_terminal.mp4 
> ### Additional Information
> * Operating System: macOS 13.5.2
> * Theia Version: 1.41.0 Blueprint app

https://github.com/eclipse-theia/theia/assets/1405703/f572a21d-6e39-480b-a7f9-69d28edb7c82

**Comment by kittaakos** [comment]
> When I resize the terminal, the width of the vertical scrollbar changes; sometimes, it's hardly visible.

It works with `Version: 1.58.0`.


https://github.com/user-attachments/assets/1831ba0b-1049-42fe-9ae7-9b2d35abfbed

https://github.com/user-attachments/assets/56348bf9-0b2a-4a44-9c88-8c9140bc5734

https://github.com/user-attachments/assets/8293b705-0cda-4654-bdcc-3b04f822262a

https://github.com/user-attachments/assets/5ffcb014-6a1c-4715-a15b-c5c7bc835171
